queries.sql

```diff
@@ -0,0 +1,10 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth < '2019-12-31';
+SELECT name FROM animals WHERE neutered = 't' AND escape_attempts < 3;
+SELECT date_of_birth FROM animals where name LIKE '%Agumon' OR name LIKE '%Pikachu';
```

Comment:
as part of the requirements, you should List the date of birth of all animals named either "Agumon" or "Pikachu" in your case you used 
```SQL
SELECT date_of_birth FROM animals where name LIKE '%Agumon' OR name LIKE '%Pikachu';
```
and that will return any value CONTAIN 'Agumon' Or 'Pikachu' , so It can return incorrect names like `ExamplePikachu ` , and the requirements say it needs to be EQUAL to exactly `Pikachu` or `Agumon` , so kindly fix it by updating your query condition  `WHERE name = 'Pikachu' OR name = 'Agumon'; `🙏

Reply:
Thank you for the feedback. The query is updated.